runner/runners/queue.go

```diff
@@ -62,13 +62,11 @@ func NewQueueRunner(
 				statusManager.UpdateService(runner.ServiceStatus{Initialized: false, Error: err})
 			} else {
 				statusManager.UpdateService(runner.ServiceStatus{Initialized: true})
+				go controller.loop()
```

Comment:
I don't think this changes - assumption is we can't start the loop unless we were initialized successfully, which needs logic in 2 different places depending on whether we had an initDoneCh